README.md

```diff
@@ -109,7 +109,8 @@
 * [graphql-java-datetime](https://github.com/donbeave/graphql-java-datetime): A set of ISO 33601, RFC 3339 compatible date time scalars for GraphQL Java
 
 ## Tools
-
+* [Firecamp](https://firecamp.io/graphql): The fastest GraphQL playground for altra modern team.
```

Comment:
I think you mean "ultra" instead of "altra"?

Reply:
Oppss... you're right